src/main/app/controller/courts/CourtTypesController.ts

```diff
@@ -14,6 +17,9 @@ export enum courtType {
 export class CourtTypesController {
 
   emptyCourtCodeErrorMsg = 'Court code is required and must be numeric and start with 1-9';
+  emptyDxCodeErrorMsg = 'Code is required for all Dx code entries.';
+  duplicatedDxCodeErrorMsg = 'All dx codes must be unique..';
```

Comment:
minor, two dots at the end, not a biggy. Prob just me being pedantic

Reply:
removed extra dot